### DISCUSSION TITLE
Free Text To Speech

### DISCUSSION THREAD

**Discussion by dewnaka7264** [community/community#136968] [Programming Help]
### Select Topic Area

Question

### Body

Hey!!! Do you guys know a way to turn text speech and get a audio file for free? A website or an application?

**Comment by ebndev** [comment]
Hi @dewnaka7264, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

**Reply by dewnaka7264** [reply]
thanks!!!!

**Reply by GeronimoKTT** [reply]
@dewnaka7264 so far the best website I know that truly give you free text to speech is [www.nemesyslabs.com](https://www.nemesyslabs.com/)
Better than 99% of other platforms in my opinion, they also give you api access.

**Comment by Giddy-K** [comment]
Hey...Here are some recommendations:
1. Natural Reader (Online & App)
2. Website: [TTSMP3](https://ttsmp3.com/)
3. App: Balabolka
4. Website: IBM Watson TTS
5. Website: Capti Voice

**Comment by guvenkarabulut** [comment]
You can use Elevenlabs

https://elevenlabs.io/
